frontend/package.json

```diff
@@ -248,5 +249,6 @@
   },
   "lint-staged": {
     "*.{js,jsx,ts,tsx,json,gql,graphql}": "eslint --color --fix"
-  }
+  },
+  "version": "0.0.0"
```

Comment:
Was this an intentional change?

Reply:
no, most probably done by `husky` ?

Reply:
but after I updated the minimist to `1.2.3` it's gone... weird